pmm-app/src/pmm-qan/panel/components/Overview/components/QanTable/Table.tsx

```diff
@@ -127,77 +128,72 @@ export const Table: FC<TableProps> = ({
     ({ index, style }) => {
       const row = rows[index];
 
       prepareRow(row);
 
       return (
-        <>
-          {index === 0
-            ? headerGroups.map((headerGroup) => (
-              <div {...headerGroup.getHeaderGroupProps()} className={cx('tr', styles.headerRow)}>
-                {headerGroup.headers.map((c, index) => {
-                  const column = c as any;
-
-                  const { HeaderAccessor } = column;
-
-                  column.Header = () => HeaderAccessor();
-                  let sorted = '';
-
-                  if (column.isSorted) {
-                    if (column.isSortedDesc) {
-                      sorted = 'desc';
-                    } else {
-                      sorted = 'asc';
-                    }
-                  }
-
-                  return (
-                    <div {...column.getHeaderProps()} className={cx('th', { [styles.rowNumberCell]: index === 0 })}>
-                      <div className={styles.headerContent}>
-                        <div className="header-wrapper">{column.render('Header')}</div>
-                        {column.sortable ? (
-                          <a className={styles.sortBy} {...column.getSortByToggleProps()} data-qa="sort-by-control">
-                            <span className={`sort-by ${sorted}`} />
-                          </a>
-                        ) : null}
-                      </div>
-                    </div>
-                  );
-                })}
-              </div>
-            ))
-            : null}
-          <div
-            {...row.getRowProps({
-              style,
-            })}
-            className={cx('tr', `tr-${row.index}`, rowClassName(row.original, row.index))}
-            onClick={() => {
-              const selectedColumn = document.querySelector(`.tr-${row.index}`);
-              const tableBody = document.querySelector('.table-wrapper .table-body');
-
-              if (selectedColumn && tableBody) {
-                setTimeout(() => {
-                  tableBody.scroll(0, (selectedColumn as HTMLElement).offsetTop - 55);
-                });
-              }
-
-              onRowClick(row);
-            }}
-          >
-            {row.cells.map((cell) => (
-              <div {...cell.getCellProps()} className={cx('td', styles.tableCell)}>
-                {cell.render('Cell')}
-              </div>
-            ))}
-          </div>
-        </>
+        <div
+          {...row.getRowProps({
+            style,
+          })}
+          className={cx('tr', `tr-${row.index}`, rowClassName(row.original, row.index))}
+          onClick={() => {
+            const selectedColumn = document.querySelector(`.tr-${row.index}`);
+            const tableBody = document.querySelector('.table-wrapper .table-body');
+
+            if (selectedColumn && tableBody) {
+              setTimeout(() => {
+                tableBody.scroll(0, (selectedColumn as HTMLElement).offsetTop - 55);
```

Comment:
It's better to use a constant here instead of that `55`, also to better explain what this line does